docs/user-guide/installation.rst

```diff
@@ -12,12 +26,15 @@ There are a number of optional dependency groups you can install for extra funct
 These can be installed using ``pip install "zarr[<extra>]"``, e.g. ``pip install "zarr[gpu]"``
 
 - ``gpu``: support for GPUs
-- ``fsspec``: support for reading/writing to remote data stores
-- ``tree``: support for pretty printing of directory trees
+- ``remote``: support for reading/writing to remote data stores
+
+Additional option dependencies include ``rich``, ``universal_pathlib``. These must be installed separately.
```

Comment:
We do have the `optional` extra that installs these two - I think either we should mention that here in the docs, or get rid of it (I have no strong feelings either way)